site/guides/enhanced-apis-nft.md

```diff
@@ -0,0 +1,119 @@
+
+We will use the Alchemy SDK Client to extend our Alchemy Provider using the provider's [`withAlchemyEnhancedApis`](/packages/aa-alchemy/provider/withAlchemyEnhancedApis.md) method. That way, our provider will have direct access to the Enhanced APIs. For the purposes of our example, we'll use the NFT API.
+
+To use the Alchemy SDK in our project directory, we'll need to install the required package:
+
+::: code-group
+
+```bash [npm]
+npm i alchemy-sdk
+```
+
+```bash [yarn]
+yarn add alchemy-sdk
+```
+
+:::
+
+Then, all we need to do is create an `alchemy` client from the Alchemy SDK, create an `AlchemyProvider` from Account Kit, and then extend the provider with functionality from the SDK client using `withAlchemyEnhancedApis`. We can get the smart account's address from the `AlchemyProvider` in order to fetch the smart account's NFT in just 1 line of code!
+
+<<< @/snippets/enhanced-apis-example/alchemy-sdk-extend/example-nft.ts
+
+:::tip Note
+Note that we must configure the Alchemy SDK client to have the same API Key and blockchain network as Alchemy Provider it is extending via `withAlchemyEnhancedApis`. This method explicitly checks this requirement and will throw an error at runtime if not satisfied.
+
+Additionally, since the Alchemy SDK client does not support JWT authentication, an `AlchemyProvider` initialized with JWTs cannot use this method. We must be initialize the provider with an API key or RPC URL.
+:::
+
+## 2. Alchemy SDK
+
+We can also use the Alchemy SDK client directly to query blockchain data for our `AlchemyProvider`.
+
+Since we still need to use the Alchemy SDK in our project directory, we'll need to install the required package:
+
+::: code-group
+
+```bash [npm]
+npm i alchemy-sdk
+```
+
+```bash [yarn]
+yarn add alchemy-sdk
+```
+
+:::
+
+Then, it's simply a matter of creating an `AlchemyProvider` from Account Kit and then creating an `alchemy` client from the Alchemy SDK. We can get the smart account's address from the `AlchemyProvider` in order to fetch the smart account's NFT in just 1 line of code!
+
+<<< @/snippets/enhanced-apis-example/alchemy-sdk/example-nft.ts
+
+:::tip Note
+This approach does not explicitly check that the Alchemy SDK client to have the same API Key and blockchain network as Alchemy Provider it is extending via `withAlchemyEnhancedApis`. Therefore, we highly recommend approach 1.
+:::
+
+There are so many more Enhanced APIs the the Alchemy SDK exposes, and can be useful for development with Account Abstraction. Try it out [here](https://github.com/alchemyplatform/alchemy-sdk-js)!
+
+## 3. Custom Viem Client
+
+One of the best parts of Account Kit is that it's build atop [viem](https://viem.sh/). This means we can leverage utility methods to easily generate a JSON-RPC client, extend it with custom functionality to call Alchemy Enhanced APIs, and expose type-safe decorators to rapidly improve development.
```

Comment:
I think asserting aa-sdk being on top of viem as one of the best parts of it is a bit debatable. We don't want to turn away customers who are not using viem (many OG).